modules/installation-uninstall-aws.adoc

```diff
@@ -7,27 +7,87 @@
 
 You can remove a cluster that you installed on Amazon Web Services (AWS).
 
+.Prerequisites
+
+* Deploy an {product-title} cluster.
+* Have a copy of the installation program that you used to deploy the cluster.
+
 .Procedure
 
-. Optionally, from the computer that you used to install the cluster, run the
-following command and record the UUID that it outputs:
+Perform the following steps on the computer that you used to install the cluster.
+
+. Obtain the cluster UUID:
+
+** If you have access to the `oc` command line interface (CLI), log in and run the
+following command:
 +
-[source,bash]
 ----
 $ oc get clusterversion -o jsonpath='{.spec.clusterID}{"\n"}' version
+
+df328c20-0dbd-4d02-a694-30949d7856e7
+----
++
+The command output is the cluster's UUID. You can use this UUID to generate a new
+`metadata.json` file for your cluster, if required. If not all of the cluster
+resources are removed from AWS when you delete the cluster, you can also use
+this UUID to locate them and remove them.
+** If you do not have access to the `oc` CLI but you know the AWS region the
```

Comment:
How about:
s/If you do not have access to the `oc` CLI /If you do not have access to the existing cluster via the `oc` CLI /